lock/src/main/java/org/infinispan/lock/impl/manager/EmbeddedClusteredLockManager.java

```diff
@@ -217,6 +227,12 @@ public void execute(Runnable runnable) {
       executor.execute(runnable);
    }
 
+
+   //Visible for testing
+   AdvancedCache<ClusteredLockKey, ClusteredLockValue> getCache() {
```

Comment:
Not sure you need this, the test could use `cache(0, ClusteredLockModuleLifecycle.CLUSTERED_LOCK_CACHE_NAME)`

Reply:
ok